pkg/cloudprovider/providers/azure/azure_backoff.go

```diff
@@ -134,6 +180,28 @@ func (az *Cloud) CreateOrUpdateSGWithRetry(service *v1.Service, sg network.Secur
 	})
 }
 
+// CreateOrUpdateLB invokes az.LoadBalancerClient.CreateOrUpdate with exponential backoff retry
+func (az *Cloud) CreateOrUpdateLB(service *v1.Service, lb network.LoadBalancer) error {
+	if az.Config.shouldOmitCloudProviderBackoff() {
+		ctx, cancel := getContextWithCancel()
+		defer cancel()
+
+		resp, err := az.LoadBalancerClient.CreateOrUpdate(ctx, az.ResourceGroup, *lb.Name, lb)
+		klog.V(10).Infof("LoadBalancerClient.CreateOrUpdate(%s): end", *lb.Name)
+		if err == nil {
+			if isSuccessHTTPResponse(resp) {
+				// Invalidate the cache right after updating
+				az.lbCache.Delete(*lb.Name)
+			} else if resp != nil {
+				return fmt.Errorf("HTTP response %q", resp.Status)
+			}
+		}
+		return err
+	}
+
+	return az.CreateOrUpdateLBWithRetry(service, lb)
```

Comment:
So `CreateOrUpdateLBWithRetry` may be changed to `createOrUpdateLBWithRetry` since it's not external now.